components/Form/form.css

```diff
@@ -0,0 +1,119 @@
+:root {
+  --primary-color: #ff6d6d;
+  --text-color: rgb(73, 73, 73);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Generally, we add this file in reset CSS so no need to write in every CSS file. just write once and import to another folder.

Reply:
Yes. I have added this by mistake